### PR TITLE
Add aETH rate calculator contract

### PR DESCRIPTION
**Pull request by skellet0r**
### What I did

Ahead of curvefi/curve-pool-registry#19, I've made a couple additions. Specifically this PR accounts for aETH related adjustments.

- Added template rate calculator for pools
- Added the aETH exchange rate calculator which conforms to the expected ABI for the new registry update
- Added to the deploy script to deploy the rate calculator of a given pool (this way rate_calculator deployment is handled at deploy time)
- Updated the respective READMEs

### How I did it

Used my l33t haX0r ski11zzz

### How to verify it

Not really much to verify but I guess you could try deploying a pool in forked-mainnet mode for fun

### CODE REVIEW

File: contracts/pools/aeth/RateCalculatorAETH.vy [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# @version 0.2.11
+"""
+@title Curve Pool Rate Calculator 
+@author Curve.Fi
+@license Copyright (c) Curve.Fi, 2021 - all rights reserved
+@notice Logic for calculating exchange rate for a pool
+@dev This contract is only a template, pool-specific logic
```
Comment (iamdefinitelyahuman) [inline contracts/pools/aeth/RateCalculatorAETH.vy:7]: This comment can be removed in the not-template version

**Review by iamdefinitelyahuman — APPROVED** [review]
Looks good!  Will deploy the AETH version so we can add in the address, and then can merge.

File: contracts/pools/aeth/RateCalculatorAETH.vy (hunk 1 above)
Comment (skellet0r) [reply contracts/pools/aeth/RateCalculatorAETH.vy:7]: oh shooot def forgot to remove that will update

File: contracts/pools/aeth/pooldata.json [hunk 2]
```diff
@@ -31,5 +31,6 @@
   ],
   "testing": {
     "initial_amount": 10000
-  }
+  },
+  "rate_calculator_address": ""
```
[the remark this answers is not in the document]
Comment (skellet0r) [reply contracts/pools/aeth/pooldata.json]: SWEET!!

**Review by iamdefinitelyahuman — APPROVED** [review]

**Pull request merged**